CHANGELOG.md

```diff
@@ -7,6 +7,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.0] - 2024-04-08
+
+### Added
+
+- add FIPS-related functionality ([#181])
+- twoilter: add build clean command ([#183])
+
+[#181]: https://github.com/bottlerocket-os/twoliter/pull/181
+[#183]: https://github.com/bottlerocket-os/twoliter/pull/183
+
+### Changed
+
```

Comment:
We're missing an important PR that switches to the "unified SDK" and is a breaking change.